samples/testapp/iosApp/TestApp/Info.plist

```diff
@@ -45,6 +45,13 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NFCReaderUsageDescription</key>
+	<string>This app uses NFC to scan identity credentials</string>
+    <key>com.apple.developer.nfc.readersession.iso7816.select-identifiers</key>
```

Comment:
nit: Indentation looks off for these lines...

Reply:
I actually think it's correct...